static/images/wifi-secure.svg

```diff
@@ -1,49 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Is this image not still used in Wi-Fi setup? https://github.com/WebThingsIO/gateway/blob/master/src/wifi-setup.js#L136

Reply:
Good catch! I was looking in the wrong place.